frontend/app/routes/public/renew/$id/adult-child/update-address.tsx

```diff
@@ -299,16 +292,6 @@ export default function RenewAdultChildUpdateAddress() {
                 errorMessage={errors?.mailingAddress}
                 required
               />
-              <InputSanitizeField
```

Comment:
Looks like the help/additional information text has also changed and mentions to include the apartment number:

![image](https://github.com/user-attachments/assets/05a87744-4186-45fa-b8e1-658e4559a96d)

We can also remove the unused i18n keys for the `label` and error message.